public/components/custom_panels/helpers/utils.tsx

```diff
@@ -479,7 +479,7 @@ export const displayVisualization = (metaData: any, data: any, type: string) =>
         query: { rawQuery: metaData.query },
         indexFields: {},
         userConfigs: mixedUserConfigs,
-        explorer: { explorerData: data, explorerFields: data.metadata.fields },
+        explorer: { explorerData: data, explorerFields: data.schema },
```

Comment:
Do we need to send multiple fields here?

Reply:
Yes it will be handled, and these will be restructured in later PRs.